mslib/mswms/mpl_vsec.py

```diff
@@ -346,3 +348,5 @@ def plot_vsection(self, data, lats, lons, valid_time, init_time,
 
             # Return the XML document as formatted string.
             return xmldoc.toprettyxml(indent="  ")
+        else:
+            raise RuntimeError
```

Comment:
The code should not be able to get here at all as wrong formats should be caught before going here.
The RuntimeError will be caught in wms.py and logged there. This will not be helpful to anyone but a developer as such a log entry is not needed.